src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/EnhancedFurnace.java

```diff
@@ -1,74 +0,0 @@
-package me.mrCookieSlime.Slimefun.Objects.SlimefunItem;
```

Comment:
You should not delete the Enhanced Furnace

Reply:
added it back